aggregate/load_aggregated.py

```diff
@@ -69,3 +69,37 @@ def initialize_dataframe_geo_index(geography, columns=[]):
     rv = pd.DataFrame(index=indicies[geography], columns=columns)
     rv.index.rename(geography, inplace=True)
     return rv
+
+
+"""this is specifically to use for housing security and quality March 4th POP data"""
+def load_clean_pop_data(ind_name_str: str) -> pd.DataFrame:
+    """Function to merge the two files for the QOL outputs and do some standard renaming. Because
+    these are QOL indicators they remain in the same csv output with columns indicating year"""
+
```

Comment:
Maybe the code to construct the regex from the mapper dictionary should live here? would DRY out all the places where we have `ind_name_str = "|".join([k for k in name_mapper.keys()])`. so this function would be passed a mapper instead of the regex string and construct the regex string here